src/scripts/main.js

```diff
@@ -1,3 +1,73 @@
 'use strict';
 
-// write code here
+const titles = [...document.querySelectorAll('thead tr th')];
+
+const nameTitle = titles[0];
+const positionTitle = titles[1];
+const ageTitle = titles[2];
+const salaryTitle = titles[3];
```

Comment:
```suggestion
const [nameTitle, positionTitle, ageTitle, salaryTitle] = titles;
```
NI
better to use destructuring